server/controllers/UserController.js

```diff
@@ -34,17 +34,18 @@ class UserController {
           message: 'Email or Username is already in use by another User.'
         });
       }
-      User
-        .create({
-          firstname,
-          lastname,
-          username,
-          email,
-          password,
-          bio
-        })
+      User.create({
+        firstname,
+        lastname,
+        username,
+        email,
+        password,
+        bio
+      })
         .then((user) => {
-          const token = jwt.sign({ userId: user.id }, secret, { expiresIn: '24h' });
```

Comment:
I think spreading the object for a single key: value in an object into a multiple line is not necessary unless we have multiple keys: values.